torch/_inductor/overrides.py

```diff
@@ -35,21 +35,25 @@ def __torch_function__(self, func, types, args=(), kwargs=None):
             config.fallback_random
             and replacements[func] in replacements_using_triton_random
         ):
-            return replacements[func](*args, **kwargs)
+            if func != torch.nn.functional.dropout or (
+                func == torch.nn.functional.dropout
+                and args[0].device != torch.device("cpu")
+            ):
```

Comment:
How about create a set to hold all the functions to fallback on cpu, e.g., `fallbacks_cpu_random`? Then, we check whether `func` is in it or not here without explicitly checking a specific function here.

Reply:
Done in [7aa97a7](https://github.com/pytorch/pytorch/pull/97002/commits/7aa97a7d8a8bcc3dd7cf40eaad52223cd3eebc45)